pre-commit.hcl

```diff
@@ -0,0 +1,18 @@
+description = "Pre-commit is a multi-language package manager for pre-commit hooks."
+binaries = ["pre-commit"]
+test = "pre-commit --version"
+source = "https://github.com/pre-commit/pre-commit/releases/download/v${version}/pre-commit-${version}.pyz"
+strip = 1
+runtime-dependencies = ["python3@3.9"]
+provides=["pre-commit"]
```

Comment:
This should only be used for "virtual" packages such as "jre", and should not match an existing "real" package. You'll have to remove it.

Reply:
Want to fix these up and merge?

Reply:
✅